src/errors.rs

```diff
@@ -35,6 +38,7 @@ error_chain! {
     foreign_links {
         EdnParseError(edn::ParseError);
         Rusqlite(rusqlite::Error);
+        UuidParseError(uuid::ParseError);
```

Comment:
Pushing the UUID errors down into being a Tolstoy error would allow us to omit this.